app/views/boarding_points/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Edit Boarding Point Information</h1>
+<%= render 'form', route: @boarding_point %>
```

Comment:
why  route: @boarding_point ?

Reply:
Thanks for the feedback vaia. It was a typo by me. Changed it to `boarding_point: @boarding_point`